Duplicati/Server/Scheduler.cs

```diff
@@ -278,7 +278,19 @@ private void Runner()
                                 {
                                     var entry = Program.DataConnection.GetBackup(id);
                                     if (entry != null)
-                                        jobsToRun.Add(Server.Runner.CreateTask(Duplicati.Server.Serialization.DuplicatiOperation.Backup, entry));
+                                    {
+                                        Dictionary<string, string> options = Duplicati.Server.Runner.GetCommonOptions(entry, Duplicati.Server.Serialization.DuplicatiOperation.Backup);
+                                        Duplicati.Server.Runner.ApplyOptions(entry, Duplicati.Server.Serialization.DuplicatiOperation.Backup, options);
+                                        if ((new Duplicati.Library.Main.Options(options)).DisableOnBattery && (Duplicati.Library.Utility.Power.PowerSupply.GetSource() == Duplicati.Library.Utility.Power.PowerSupply.Source.Battery))
+                                        {
+                                            // TODO: Can we log this somewhere?  It would be nice to provide a way for the
+                                            // user to know why the backup wasn't run.

```

Comment:
I've made several unsuccessful attempts at logging this case.  Any ideas?  It would be nice to provide a way for the user to know why the backup wasn't run.

Reply:
You can write to `Duplicati.Library.Logging.Log.WriteInformationMessage()`, we just need to make sure it doesn't spam the log.

I am rewriting the log system, so we can present it in a nice way later.

Reply:
Thanks.  I had tried that earlier but was looking at the wrong logs when testing.  Added the logging statement in revision f3ca95763f16dfe939dbfe95fa9baeb1c49db219.